index.js

```diff
@@ -50,10 +76,33 @@ const formButtons = [
     view: "button",
     value: "Add new",
     css: "webix_primary",
+    click: function () {
+      var values = $$("films-form").getValues();
+      const isValid = $$("films-form").validate();
```

Comment:
`var` => `const`
Also, move `$$("films-form")` (and anything that is called more than once in a function) to a variable